.eslintrc.js

```diff
@@ -17,14 +17,6 @@ module.exports = {
     plugins: [
         'vue',
     ],
-    // Check if imports actually resolve
-    settings: {
-        'import/resolver': {
-            webpack: {
-                config: 'build/webpack.client.config.js'
```

Comment:
This was never quite right, since it points to the base config in vue-ssr-build, but that doesn't export a config - it exports a function that returns a config when given proper options.  `eslint-plugin-import` is smart enough to call the function, but since it doesn't pass any options now everything was accurate.  Specifically, `config.rootDir` wasn't defined, so that broke a lot of our aliases and webpacks ability to transform import paths.

So, instead, we just remove this and it's associated rules entirely and leave them up to the client app - that way they can point to their higher level webpack boncifg.  They were both being overriden in SlipStream as it was, and if they weren't being overridden in Typhoona, they probably didn't work right either way :).